components/composer/src/components/Attachment.svelte

```diff
@@ -79,31 +76,38 @@
     }
   }
 
-  .LoadingIcon {
-    fill: var(--composer-icons-color, #666774);
-    width: 15px;
-    height: 15px;
-    animation: rotate 0.5s infinite linear;
-  }
-  .CloseIcon {
-    fill: var(--composer-icons-color, #666774);
-    width: 10px;
-    height: 10px;
+  @media #{$desktop} {
+    .file-info {
+      flex-direction: row;
+      align-items: center;
+      max-width: 90%;
+    }
+    .file-item__size {
+      margin: 0 0 0 5px;
+    }
   }
 </style>
 
 {#if attachment}
   <div class="wrapper">
     <div class="file-item">
       <div class="file-info">
-        <div class="ellipsis">{attachment.filename}</div>
-        <span class="file-item__size">{prettySize(attachment.size)}</span>
+        {#if !attachment.error}
+          <div>{attachment.filename}</div>
+          <span class="file-item__size">{prettySize(attachment.size)}</span>
+        {/if}
       </div>
       <div class="file-info__right">
         {#if attachment.loading}
-          <LoadingIcon class="LoadingIcon" />
+          <LoadingIcon
+            style="fill: var(--composer-icons-color, #666774); width: 15px; height: 15px; animation: rotate 0.5s infinite linear;"
+          />
+        {/if}
+        {#if attachment.error}
+          <span class="file-info__error"
+            >Error: Please try attaching the file again.</span
```

Comment:
Oh no! The closing tag bracket issue thing resurfaced 😞

Reply:
:<